Automattic/portkey-android
# Update configure plugin to 0.6.5

Updates configure plugin to `0.6.5` and the `.configure` hash to latest to include change to the secret `gradle.properties` file that enables non transitive resources.

**To Test**

* Make sure to delete the `vendor` folder first. There is still a bug in the configure plugin where it doesn't update the binary if it's already downloaded.
* Run `./gradlew applyConfiguration`
* Verify that `android.nonTransitiveRClass=true` is included in the new `gradle.properties` file.
